MvvmCross-AndroidSupport/MvvmCross.Droid.Support.Fragment/MvxCachingFragmentActivity.cs

```diff
@@ -263,6 +263,13 @@ protected virtual void ShowFragment(string tag, int contentId, Bundle bundle, bo
 			{
 				((Android.Support.V4.App.Fragment)fragInfo.CachedFragment).Arguments.Clear();
 				((Android.Support.V4.App.Fragment)fragInfo.CachedFragment).Arguments.PutAll(bundle);
+
+                var childViewModelCache = Mvx.GetSingleton<IMvxChildViewModelCache>();
+                if (childViewModelCache.Exists(fragInfo.CachedFragment.ViewModel.GetType()))
+                {
+                    fragInfo.CachedFragment.ViewModel = childViewModelCache.Get(fragInfo.CachedFragment.ViewModel.GetType());
+                    childViewModelCache.Remove(fragInfo.CachedFragment.ViewModel.GetType());
```

Comment:
Throw `fragInfo.CachedFragment.ViewModel.GetType()` into a variable so it's not evaluated each time.